repositories/src/main/scala/cromwell/pipeline/datastorage/dto/ProjectConfiguration.scala

```diff
@@ -6,16 +6,24 @@ import ProjectFile.pathFormat
 import org.mongodb.scala.bson.collection.immutable.Document
 import play.api.libs.json.{ Json, OFormat }
 
-case class ProjectFileConfiguration(path: Path, inputs: List[FileParameter])
+case class ProjectFileConfiguration(path: Path, inputs: Set[FileParameter])
 
 object ProjectFileConfiguration {
   implicit val projectFileConfigurationFormat: OFormat[ProjectFileConfiguration] = Json.format
+
+  def toMap(config: ProjectFileConfiguration): Map[Path, Map[String, TypedValue]] =
+    Map(
+      config.path -> config.inputs.flatMap(fileParameter => Map(fileParameter.name -> fileParameter.typedValue)).toMap
+    )
```

Comment:
I don't think this is an appropriate place for such method (same for `fromMap`). It solves a very specific problem and it's better to put it where you actually need it.

Reply:
Moved.